### PR TITLE
#144#145#147#149#151 updating bios and team page based on main

### PR DESCRIPTION
**Pull request by michellemcgowan**
- #151 - Gretchen
  - updated bio and title
- #149 - Sami
  - updated bio
- #145 - John
  - updated bio
- #144 - Natalie
  - updated bio and title 

- #147 is going to be removed in June

### CODE REVIEW

File: TeamPage.html [hunk 1]
```diff
@@ -1267,6 +1253,10 @@ <h6 id="contactID" class="h6 text-uppercase mb-4 font-weight-bold">Contact</h6>
             <p>
               <i class="fas fa-envelope mr-3" aria-label="Envelope Icon"></i><a aria-describedby="contactID"
                 href="mailto:gretchen.andrew@unlv.edu">gretchen.andrew@unlv.edu</a><br>
+              <i class="fas fa-phone mr-3" aria-label="Phone Icon"></i><a aria-describedby="contactID"
+                href="tel:7028951437">(702)-895-1437 (Phone)</a><br>
+              <i class="fas fa-print mr-3" aria-label="Fax Icon"></i><a aria-describedby="contactID"
+                href="tel:7028954064">(702)-895-4064 (Fax)</a><br>
```
Comment (Kelvyn-Meyers) [inline TeamPage.html:1259]: Was this added by accident or did Natalie request we re-add these to the website?

File: TeamPage.html [hunk 2]
```diff
@@ -947,14 +933,14 @@ <h2>Meet Our Team</h2><br>
...
+                      <p class="text-justify">Natalie Renkes joined the UNLV-FIA group in May 2020. She received her M.S. degree in Geoscience from UNLV in May of 2023. She supervises student interns/staff members, is project manager for the DATIM, EVALIDator, Sample Nonresponse Bias in Estimations, and Science Strategy projects; and performs other duties based on FIA National Operations priorities. As a member of the FIA Section 508 Program Development Team, she is also working to ensure our group's compliance with federal accessibility laws. Outside of UNLV-FIA, Natalie’s research interests focus on naturally-occurring asbestos and its health implications within metropolitan Las Vegas. In her spare time, Natalie enjoys playing The Legend of Zelda: Breath of the Wild, hiking/camping, and is an avid reader. Her favorite book series is Game of Thrones (she hasn’t seen the show, don’t ruin the ending).
```
Comment (Kelvyn-Meyers) [inline TeamPage.html:943]: Just to match Natalie's exact wording, I think this bio should end with an exclamation mark. But if you disagree, let me know why and we can just ignore this!

**Review by Kelvyn-Meyers — CHANGES_REQUESTED** [review]
Hi @michellemcgowan,

I think we're almost good to go here! I just left a couple notes regarding some punctuation in one of the bios and some accidentally added contact information in the footer. Please see my in-line comments below and let me know if you have any questions!

File: TeamPage.html (hunk 1 above)
Comment (michellemcgowan) [reply TeamPage.html:1259]: OMG twas an accident! Apologies! I don't think I pulled down the new changes! Fixing it

**Pull request merged**